include/rk_types.h

```diff
@@ -3,6 +3,7 @@
 #include <stdint.h>
 #include <stddef.h>
 #include <macros.h>
+#include <rk_common.h>
```

Comment:
This header is for revolution types and can be included in low level game agnostic stuff, e.g. system libraries. It's better to just include "rk_common.h" directly wherever it is used